p9Layouts/axonePnorLayout_64.xml

```diff
@@ -362,4 +362,14 @@ Layout Description
         <readOnly/>
         <ecc/>
     </section>
+    <section>
+        <description>Hostboot Runtime Proxy (32KB)</description>
+        <eyeCatch>HBRT_PROXY</eyeCatch>
```

Comment:
8 chars or less is ideal if the partition were to ever be loaded as a LID or something, but that likely not being the case, should be ok.